db/data_migration/20170906102634_redirect_cips.rb

```diff
@@ -0,0 +1,64 @@
+class RedirectCIPs
+  def redirect_all
+    about_us = CorporateInformationPageType::AboutUs
+
+    published_about_pages = CorporateInformationPage
+      .published
```

Comment:
There are a few `withdrawn` ones too. Do we need to include those? There are 5 as far as I can see e.g. https://www.gov.uk/government/organisations/oil-and-gas-authority/about/about